exporters/autoexport/metrics.go

```diff
@@ -148,6 +195,10 @@ func init() {
 
 		return readerWithServer{lis.Addr(), reader, &server}, nil
 	})
+
+	RegisterMetricProducer("prometheus", func(ctx context.Context) (metric.Producer, error) {
```

Comment:
Should we register a "none" producer?  It would be used if someone didn't want to use the fallback producer.

Reply:
Good point, done!